src/components/SearchResult/SearchResult.tsx

```diff
@@ -29,12 +30,18 @@ const SearchResult = ({ bib }: SearchResultProps) => {
         <RCLink href={`${PATHS.BIB}/${bib.id}`}>{bib.title}</RCLink>
       </CardHeading>
       <CardContent>
-        <Box sx={{ p: { display: "inline", marginRight: "s" } }}>
+        <Box sx={{ p: { display: "inline", marginRight: "s" } }} mb="m">
           {bib.materialType && <Text>{bib.materialType}</Text>}
           {bib.publicationStatement && <Text>{bib.publicationStatement}</Text>}
           {bib.yearPublished && <Text>{bib.yearPublished}</Text>}
           <Text>{bib.itemMessage}</Text>
         </Box>
+        {bib.electronicResources && (
```

Comment:
In the Vercel preview, all bib search results are displaying the `ElectronicResourcesLink` and that's not what true for the same searches in production. Something seems to be off.

Reply:
I think the issue cab be addressed here https://github.com/NYPL/research-catalog/blob/5f63115adf9986c4f56817090f353cccba19cd51/src/models/SearchResultsBib.ts#L35
There should be a check for `electronicResources?.length` because an empty array is truthy.

Reply:
Just added a helper function hasElectronicResources that checks for the length. This resolves the issue.